db/data.py

```diff
@@ -187,6 +189,22 @@ def _insert_lowlevel_json(connection, ll_id, data_json, data_sha256, version_id)
                                    "data_sha256": data_sha256,
                                    "version": version_id})
 
+    def _get_submission_offset(connection, mbid):
+        """ Get highest existing submission offset for mbid, then increment """
+        query = text("""
+            SELECT MAX(submission_offset)
+              FROM lowlevel
+             WHERE gid::text = :mbid
+        """)
+        result = connection.execute(query, {"mbid": mbid})
+
+        row = result.fetchone()
+        if row[0] is not None:
```

Comment:
if you give a column alias in the sql statement then you can use `row['max_offset']` here to make it a bit more explicit what we're reading